tests/conftest.py

```diff
@@ -135,3 +136,11 @@ def disable_query_cache() -> Generator[None, None, None]:
     state.set_configs({"use_cache": 0, "use_readthrough_query_cache": 0})
     yield
     state.set_configs({"use_cache": cache, "use_readthrough_query_cache": readthrough})
+
+
+@pytest.fixture
+def errors_table_name():
+    if "distributed" in os.environ["SNUBA_SETTINGS"]:
+        return "errors_dist"
+    else:
+        return "errors_local"
```

Comment:
I don't think we need a fixture for this - there is already the `schema.get_table_name()` function which will pick the right name depending on what mode you're in. This will work for all table names, not just the errors one, and does not rely on the presence of this environment variable containing the string "distributed".

Reply:
Oh right! Thank you for pointing this out.